### PR TITLE
Full review

### PR DESCRIPTION
**Pull request by VictoriaWika**

### CODE REVIEW

File: Plan.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# **Pairings**
+
+# **Komponenten**
```
Comment (aseeba) [inline Plan.md:3]: Well done! I like plans ;-)

File: components/header-sec.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  top: 22px;
+  left: 0;
+  z-index: -1;
+}
+
+.headersec__subtitle {
+  text-align: center;
+  line-height: 1.5;
+  margin-top: 0;
+}
```
Comment (aseeba) [inline components/header-sec.css:28]: An extra component for the changed header status is perhaps a little cumbersome and could be solved in a shorter way. We didn't quite understand why there is an extra component for this. Let's talk about it if you're up for another option. :-)

File: components/header-sec.stories.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import './header-sec.css'
+import kopf2 from './header-sec.html'
```
Comment (BerePendzialek) [inline components/header-sec.stories.js:2]: I would stick to English. Just for clearness and language consistency in the whole proyect.

File: components/buttons.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.buttons__energylevel-blue {
+  background: var(--color-darkblue);
+  color: var(--color-white);
+  padding: 2px 9.5px;
+  border-radius: 3px;
+  font-size: var(--font-size-small);
+}
+
```
Comment (AnnekeSchuette) [inline components/buttons.css:58]: Diese Klassen sind so spezifisch für den Bereich "Energylevel" und tauchen sonst nirgends auf, sodass man sie vielleicht besser in der Datei energylevel.css statt in der Komponente unterbringen sollte. 

Die Farben ließen sich zudem auch gut über eine modification-class am Button-Element (siehe Kommentar weiter oben) realisieren.

File: components/header.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
+.header {
+  position: fixed;
+  width: 100vw;
+  background-color: var(--color-eggshell);
+  z-index: 1;
```
Comment (aseeba) [inline components/header.css:5]: But the rest is solved consistently in BEM. I really like :-)

File: components/energylevel.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    <li>10:00</li>
+    <li>15:30</li>
+    <li>16:15</li>
+    <li>18:25</li>
+    <li>19:00</li>
+  </ul>
+</section>
+<section class="buttons__positioning margin-t10 margin-b30">
+  <span class="buttons__energylevel-blue">Other Students</span>
+  <span class="buttons__energylevel-orange">You</span>
```
Comment (laurastrehlow) [inline components/energylevel.html:44]: Class-Benennung evtl. irreführend, da eigentlich kein Button. 
```suggestion
<section class="legend margin-t10 margin-b30">
  <span class="legend__energylevel-blue">Other Students</span>
  <span class="legend__energylevel-orange">You</span>
```

File: components/header.stories.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import './header.css'
+import kopf from './header.html'
+
+export default {
+  title: 'Components',
```
Comment (BerePendzialek) [inline components/header.stories.js:5]: Here, I would suggest to add title and after that just html for the component. Something like that:

```suggestion
  title: 'Components',
  
  
  import './header.css'
import html from './header.html'

export default {
  title: 'Header',
  component: html,
}

export const base = () => html

  
```

File: components/buttons.css [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+.buttons__100,
+.buttons__75,
+.buttons__50,
+.buttons__25 {
+  padding: 12px;
+  border-radius: 5px;
+  border: none;
+  text-decoration: none;
+  text-align: center;
+}
```
Comment (AnnekeSchuette) [inline components/buttons.css:16]: Eine allgemeine Button-Klasse, z.B. "button" mit den Eigenschaften zu verwenden, wäre eine schlankere Lösung. Die Abweichungen (color, background) könnten dann per modification-Klasse umgesetzt werden, die die Farben aufgreifen (anstelle auf die Verwendung in der App hinweisen zu lassen). 

Das HTML würde für einen Button dann bspw. so aussehen:
`<button class="button button--darkorange"></button>`

File: components/navigation.css [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+  left: 20px;
+  right: 20px;
+}
+
+.navigation a {
+  text-decoration: none;
+  line-height: 0;
+}
+
+.navigation__icon-inactive {
```
Comment (BerePendzialek) [inline components/navigation.css:20]: To follow BEM, I would change to 
```suggestion
.navigation__icon-inactive {

.navigation__icon--inactive 

```

File: components/navigation.css [hunk 10]
```diff
@@ -0,0 +1,23 @@
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  background-color: var(--color-white);
+  padding: 12px 29px;
```
Comment (aseeba) [inline components/navigation.css:6]: I would suggest the following because it is a little more responsive to the different heights of the icons and unifies it better:
```suggestion
  padding: 10px 30px;
```

File: components/journal-form.html [hunk 11]
```diff
@@ -0,0 +1,55 @@
+<h2>TODAY, 12.07.2020</h2>
```
Comment (laurastrehlow) [inline components/journal-form.html:1]: Fließtext nicht in uppercase schreiben, lieber eine class vergeben und im CSS "text-transform: uppercase;" verwenden.

File: components/buttons.stories.js [hunk 12]
```diff
@@ -0,0 +1,8 @@
+import './buttons.css'
+import knopf from './buttons.html'
```
Comment (AnnekeSchuette) [inline components/buttons.stories.js:2]: Man sollte nach Möglichkeit keine Sprachen im funktionalen Code mischen (internationale Zusammenarbeit, Vermeidung von Missverständnissen usw.) => "knopf"

File: components/navigation.css [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+  border-radius: 10px;
+  box-shadow: 0 2px 11px var(--shadow-grey);
```
Comment (aseeba) [inline components/navigation.css:8]: I think the shadow is medium blue with the opacity of 20% instead of grey, like this: 
```suggestion
--shadow-dim-blue: 0 2px 11px var(--color-dim-blue-op-20);
/* (--color-dim-blue-op-20: rgb(117, 137, 162, 0.2); */
```

File: components/header-sec.css [hunk 14]
```diff
@@ -0,0 +1,28 @@
+.headersec {
+  position: fixed;
+  width: 100vw;
+  z-index: 1;
+  background-color: var(--color-eggshell);
+}
```
Comment (laurastrehlow) [inline components/header-sec.css:6]: Identisch mit .header => zusammenführen oder eine class löschen

File: journal-entry.html [hunk 15]
```diff
@@ -0,0 +1,119 @@
...
+      <img
+        class="header__background"
+        src="/icons/background_shape_1.svg"
+        alt=""
+      />
+      <h1 class="header__title">Journal</h1>
+    </header>
+
+    <main class="app-grid">
+      <h2>TODAY, 12.07.2020</h2>
```
Comment (laurastrehlow) [inline journal-entry.html:20]: Fließtext nicht in uppercase schreiben, lieber eine class vergeben und im CSS "text-transform: uppercase;" verwenden.

File: journal-form.html [hunk 16]
```diff
@@ -0,0 +1,119 @@
...
+      <img
+        class="header__background"
+        src="/icons/background_shape_1.svg"
+        alt=""
+      />
+      <h1 class="header__title">Journal</h1>
+    </header>
+
+    <main class="app-grid">
+      <h2>TODAY, 12.07.2020</h2>
```
Comment (laurastrehlow) [inline journal-form.html:20]: Fließtext nicht in uppercase schreiben, lieber eine class vergeben und im CSS "text-transform: uppercase;" verwenden.

File: components/code-buddy.html [hunk 17]
```diff
@@ -0,0 +1,12 @@
+<section class="teams margin-t34">
+  <ul class="teams__list">
+    <li class="teams__items code-buddy">
```
Comment (BerePendzialek) [inline components/code-buddy.html:3]: I would specify more what exactly this class is about. For someone external, it will be hard to follow what a code-buddy is doing.

```suggestion
    <li class="teams__items code-buddy">
    
    code-buddy-container 
    
```
This can also be a utility class, right?

File: components/energylevel.css [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+  height: 175px;
+  background: var(--color-darkorange);
+  border-radius: 50px 50px 0 0;
+}
+.energylevel__bars-blue-1 {
+  width: 12px;
+  height: 154px;
+  background: var(--color-darkblue);
+  border-radius: 50px 50px 0 0;
+}
```
Comment (AnnekeSchuette) [inline components/energylevel.css:79]: Sich wiederholende Anweisungen wie width, background und border-radius könnte man besser in einer allgemeinen Klasse, z.B. "energylevel__bar" zusammenfassen, um Code zu reduzieren.

**Review by AnnekeSchuette — COMMENTED** [review]
Very nice work! Made some suggestions specially for class names and structure.

File: components/energylevel.html (hunk 6 above)
Comment (luziehtan) [reply components/energylevel.html:44]: Gute Anmerkung - hatten wir so übernommen, weil es ja an sich das gleiche Styling ist!!

File: Plan.md [hunk 19]
```diff
@@ -0,0 +1,21 @@
+# **Pairings**
```
Comment (BerePendzialek) [inline Plan.md:1]: Great planning!

File: components/navigation.css [hunk 20]
```diff
@@ -0,0 +1,23 @@
...
+  right: 20px;
+}
+
+.navigation a {
+  text-decoration: none;
+  line-height: 0;
+}
+
+.navigation__icon-inactive {
+  filter: invert(57%) sepia(9%) saturate(959%) hue-rotate(173deg)
```
Comment (aseeba) [inline components/navigation.css:21]: It only works exceptionally because the shade is so well chosen, but here you could also achieve the same effect with the opacity, e. g. opacity: 0.5;

File: components/header.css [hunk 21]
```diff
@@ -0,0 +1,20 @@
+.header {
+  position: fixed;
+  width: 100vw;
+  background-color: var(--color-eggshell);
```
Comment (aseeba) [inline components/header.css:4]: I think the background-color is not used and only served as a placeholder, right? The graphic with the wave is used here.

File: components/navigation.html [hunk 22]
```diff
@@ -0,0 +1,7 @@
+<nav class="navigation">
+  <a href=""><img src="../icons/DashboardIcon.svg" alt="Dashboard" /></a>
+  <a href=""><img src="../icons/BuddyIcon.svg" alt="Buddy" /></a>
+  <a href=""><img src="../icons/TeamIcon.svg" alt="Team" /></a>
+  <a href=""><img src="../icons/EnergyIcon.svg" alt="Energy" /></a>
+  <a href=""><img src="../icons/JournalIcon.svg" alt="Journal" /></a>
```
Comment (aseeba) [inline components/navigation.html:6]: Is there a reason why the links to the pages are not yet in place?
```suggestion
 You could already insert the links for href-attribute here:
   <a href=""><img src="../icons/DashboardIcon.svg" alt="Dashboard" /></a>
  <a href="../index.html"><img src="../icons/BuddyIcon.svg" alt="Buddy" /></a>
  <a href="../buddies.html"><img src="../icons/TeamIcon.svg" alt="Team" /></a>
  <a href="../energy.html"><img src="../icons/EnergyIcon.svg" alt="Energy" /></a>
  <a href="../journal.html"><img src="../icons/JournalIcon.svg" alt="Journal" /></a>
```

File: components/header-sec.stories.js (hunk 3 above)
Comment (luziehtan) [reply components/header-sec.stories.js:2]: Valid point - we did this to try something on Storybook to make it work in a certain way. Definitely should change it properly!

File: styles/_utilities.css [hunk 23]
```diff
@@ -0,0 +1,42 @@
...
+  color: var(--color-lightblue);
+}
+
+.journal-heading {
+  line-height: 1.333;
+}
+
+.comprehension {
+  letter-spacing: -0.26px;
+}
```
Comment (aseeba) [inline styles/_utilities.css:12]: Utility classes should be more descriptive so that their function can be understood quickly, such as margin-t10 (or mt-10). And i think the marked classes are better kept in the components? Or is there a reason why these are included in the utility classes that i don't yet understand?

**Review by aseeba — COMMENTED** [review]
I hope my comments can help a little. If something is unclear, let's talk about it :-)

File: code-buddy.html [hunk 24]
```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Student Companion App</title>
+  </head>
+  <body>
+    <header class="headersec">
```
Comment (macfrei) [inline code-buddy.html:10]: Why not use `header` as class name? I think it would suite better than `headersec`.
```suggestion
    <header class="header">
```

File: code-buddy.html [hunk 25]
```diff
@@ -0,0 +1,161 @@
...
+      <img
+        class="headersec__background"
+        src="/icons/background_shape_1.svg"
+        alt=""
+      />
+      <h1 class="headersec__title">Code Buddys</h1>
+      <h3 class="headersec__subtitle">on Monday &#8212; 18.07.2020</h3>
```
Comment (macfrei) [inline code-buddy.html:17]: Hierarchically this `h3`-element should be a `h2`. If this is too big, then you should change the styling via CSS. 
```suggestion
      <h2 class="headersec__subtitle">on Monday &#8212; 18.07.2020</h2>
```

File: code-buddy.html (hunk 24 above)
Comment (macfrei) [reply code-buddy.html:10]: As I saw in your other HTML documents, you do have another component which has the class `header`. I am wondering why it is necessary to have two header-components and distinguish them. As far as I am concerned the only difference between the two components is the additional subtitle and the styles are the same. In this case I would suggest to only have one class `header` and then add the `header__subtitle` class if needed. 

If you have questions about that or if that is not clear, please ask me.

File: code-buddy.html [hunk 26]
```diff
@@ -0,0 +1,161 @@
...
+    </header>
+
+    <main class="app-grid">
+      <section class="teams margin-t34">
```
Comment (macfrei) [inline code-buddy.html:21]: It is a bit confusing that the class name of this component is `teams` as we are in `code-buddy.html`. I know that the styles for both components are almost identical, however I personally would separate them and distinguish those two.
```suggestion
      <section class="code-buddy margin-t34">
```

File: code-buddy.html [hunk 27]
```diff
@@ -0,0 +1,161 @@
...
+        <ul class="teams__list">
+          <li class="teams__items code-buddy">
```
Comment (macfrei) [inline code-buddy.html:23]: The additional `code-buddy`-class here is the reason why I would separate the styles of this list.
```suggestion
          <li class="code-buddy__name">
```

File: code-buddy.html [hunk 28]
```diff
@@ -0,0 +1,161 @@
...
+          <li class="teams__items">Jerry Erbs/Lene Frei</li>
+        </ul>
+      </section>
+    </main>
+    <nav class="navigation">
+      <a href="/index.html"
+        ><img
+          src="./icons/DashboardIcon.svg"
+          alt="Dashboard"
+          class="navigation__icon-inactive"
```
Comment (macfrei) [inline code-buddy.html:123]: The correct usage of BEM would be:
```suggestion
          class="navigation__icon--inactive"
```

File: components/app-grid.css [hunk 29]
```diff
@@ -0,0 +1,3 @@
+.app-grid {
+  display: inline-block;
```
Comment (macfrei) [inline components/app-grid.css:2]: What is the reason to set the display to `inline-block`? Also, the class name is really confusing since I would expect `display: grid;` here instead.

File: components/app-grid.stories.js [hunk 30]
```diff
@@ -0,0 +1,8 @@
+import './app-grid.css'
+import grid from './app-grid.html'
+
+export default {
+  title: 'Components',
+}
+
+export const Geruest = () => grid
```
Comment (macfrei) [inline components/app-grid.stories.js:8]: Please keep to English and the same here; this is not a grid. Please reconsider :)

File: components/buttons.css [hunk 31]
```diff
@@ -0,0 +1,79 @@
+.buttons {
```
Comment (macfrei) [inline components/buttons.css:1]: This name is confusing as this is a section and not a button. How about:
```suggestion
.energy-rating {
```
or something like that?

File: components/buttons.css [hunk 32]
```diff
@@ -0,0 +1,79 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
```
Comment (macfrei) [inline components/buttons.css:4]: Please be careful with this style as it is not supported by every browser. I think a nicer solution would be to set:
```css
display: grid;
grid-gap: 10px;
```

File: components/buttons.css [hunk 33]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.buttons__100,
+.buttons__75,
+.buttons__50,
+.buttons__25 {
```
Comment (macfrei) [inline components/buttons.css:10]: I agree with Annekes comment, the different background-colors would be modifications of this button. 
```suggestion
.energy-rating__button {
```

File: components/buttons.css [hunk 34]
```diff
@@ -0,0 +1,79 @@
...
+  padding: 12px;
+  border-radius: 5px;
+  border: none;
+  text-decoration: none;
+  text-align: center;
+}
+
+.buttons__100 {
```
Comment (macfrei) [inline components/buttons.css:18]: This would then be:
```suggestion
.energy-rating__button--darkorange {
```

File: components/buttons.css [hunk 35]
```diff
@@ -0,0 +1,79 @@
...
+  background: var(--color-lightorange);
+  color: var(--color-darkorange);
+}
+
+.buttons__25 {
+  background: var(--color-verylightorange);
+  color: var(--color-darkorange);
+}
+
+.buttons__positioning {
```
Comment (macfrei) [inline components/buttons.css:38]: Is this a part of this `energy-rating`-component? Or where does this apply to?

File: components/buttons.css [hunk 36]
```diff
@@ -0,0 +1,79 @@
...
+  text-align: end;
+}
+
+.buttons__energylevel-orange {
```
Comment (macfrei) [inline components/buttons.css:42]: This should be in it's own component `energylevel` as the class name suggests. With the buttons I think you got confused what components are and what not 🙂.

File: components/buttons.stories.js (hunk 12 above)
Comment (macfrei) [reply components/buttons.stories.js:2]: Agreed.

File: components/code-buddy.css [hunk 37]
```diff
@@ -0,0 +1,15 @@
+.code-buddy {
+  position: relative;
+}
+
+.margin-t34 > .teams:first-child {
```
Comment (macfrei) [inline components/code-buddy.css:5]: Why is this styling in `code-buddy.css`?

File: components/code-buddy.html (hunk 17 above)
Comment (macfrei) [reply components/code-buddy.html:3]: I kind of agree with Berenice and already commented on this earlier. I would not make it a utility class but it's own component.

File: components/energylevel.css [hunk 38]
```diff
@@ -0,0 +1,147 @@
...
+  grid-template-areas:
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    '.           time time time time time';
+}
+
+.energylevel h3 {
```
Comment (macfrei) [inline components/energylevel.css:16]: Why not give this heading a class name to be more specific?
```suggestion
.energylevel__heading {
```

File: components/energylevel.css [hunk 39]
```diff
@@ -0,0 +1,147 @@
...
+  z-index: -1;
+}
+
+.energylevel__percentage {
+  grid-area: percentage;
+  height: 208px;
+  width: 32px;
+  display: flex;
+  flex-direction: column;
+  align-items: space-around;
```
Comment (macfrei) [inline components/energylevel.css:38]: The property `space-around` does not exist for `align-items` and you could just delete it.
```suggestion
  
```

File: components/energylevel.html (hunk 6 above)
Comment (macfrei) [reply components/energylevel.html:44]: Very good point. Maybe:
```suggestion
<section class="energylevel__legend margin-t10 margin-b30">
  <span class="energylevel__legendbutton--blue">Other Students</span>
  <span class="energylevel__legendbutton--orange">You</span>
```

File: components/energylevel.stories.js [hunk 40]
```diff
@@ -0,0 +1,8 @@
+import './energylevel.css'
+import level from './energylevel.html'
```
Comment (macfrei) [inline components/energylevel.stories.js:2]: Please keep the name. `level` is not a good name as it is much more than this.
```suggestion
import energylevel from './energylevel.html'
```

File: components/header-sec.css [hunk 41]
```diff
@@ -0,0 +1,28 @@
+.headersec {
+  position: fixed;
```
Comment (macfrei) [inline components/header-sec.css:2]: I would add a `top: 0;` to these styles so that the header is always at the top.

File: components/header-sec.css [hunk 42]
```diff
@@ -0,0 +1,28 @@
...
+  width: 100vw;
+  z-index: 1;
+  background-color: var(--color-eggshell);
+}
+.headersec__title {
+  font-size: var(--font-size-big);
+  font-weight: bold 700;
+  line-height: 1.125;
+  text-transform: uppercase;
```
Comment (macfrei) [inline components/header-sec.css:11]: Very good 👍

File: components/header-sec.css [hunk 43]
```diff
@@ -0,0 +1,28 @@
+.headersec {
+  position: fixed;
+  width: 100vw;
+  z-index: 1;
+  background-color: var(--color-eggshell);
```
Comment (macfrei) [inline components/header-sec.css:5]: The background-image already has this color, right? Why do you give it to the header as well?

File: components/header-sec.css [hunk 44]
```diff
@@ -0,0 +1,28 @@
...
+  line-height: 1.125;
+  text-transform: uppercase;
+  text-align: center;
+  margin-bottom: 0;
+  margin-top: 20px;
+}
+
+.headersec__background {
+  position: absolute;
+  top: 22px;
```
Comment (macfrei) [inline components/header-sec.css:19]: If you get rid of this and of the background-color in the `.header`-class, then it should be as you wanted it to be 🙂

File: components/header.stories.js [hunk 45]
```diff
@@ -0,0 +1,8 @@
+import './header.css'
+import kopf from './header.html'
```
Comment (macfrei) [inline components/header.stories.js:2]: Again, please stick to English.

File: components/journal-form.css [hunk 46]
```diff
@@ -0,0 +1,34 @@
+.journal-form {
+  display: block;
```
Comment (macfrei) [inline components/journal-form.css:2]: Why do you set this style? I don't think you will need it.

File: components/journal-form.css [hunk 47]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.journal-form-heading {
```
Comment (macfrei) [inline components/journal-form.css:5]:
```suggestion
.journal-form__heading {
```

File: components/journal.html [hunk 48]
```diff
@@ -0,0 +1,51 @@
+<div class="buttons__center margin-t10">
```
Comment (macfrei) [inline components/journal.html:1]: Maybe:
```suggestion
<div class="journal__button margin-t10">
```

File: components/journal.html [hunk 49]
```diff
@@ -0,0 +1,51 @@
...
+  <a
+    href="/journal-form.html"
+    class="buttons__100 buttons__padding buttons__shadow"
```
Comment (macfrei) [inline components/journal.html:4]:
```suggestion
    class="journal__link"
```

File: components/navigation.css [hunk 50]
```diff
@@ -0,0 +1,23 @@
...
+  padding: 12px 29px;
+  border-radius: 10px;
+  box-shadow: 0 2px 11px var(--shadow-grey);
+  position: fixed;
+  bottom: 20px;
+  left: 20px;
+  right: 20px;
+}
+
+.navigation a {
```
Comment (macfrei) [inline components/navigation.css:15]: Maybe give a class here as well?
```suggestion
.navigation__link {
```

File: components/navigation.stories.js [hunk 51]
```diff
@@ -0,0 +1,8 @@
+import './navigation.css'
+import menu from './navigation.html'
```
Comment (macfrei) [inline components/navigation.stories.js:2]: Why not stick to the name `navigation`?

File: styles/_base.css [hunk 52]
```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
```
Comment (macfrei) [inline styles/_base.css:6]: You set the `header` and `navigation` to `position: fixed;`. Because of this, you don't need a grid. At the moment it doesn't do anything in your code. Please check and either use the grid or delete it.

File: styles/_base.css [hunk 53]
```diff
@@ -0,0 +1,51 @@
...
+
+main {
+  margin: 81px 30px;
+}
+
+button {
+  font-size: 1em;
+}
+
+.energylevel ul {
```
Comment (macfrei) [inline styles/_base.css:43]: Why is this in the `base.css`?

**Review by macfrei — COMMENTED** [review]
Good job everyone! I added a few comments you might want to consider. If you have any questions about it, please feel free to ask me about it! 🚀
